app/kudos/models.py

```diff
@@ -0,0 +1,161 @@
+# -*- coding: utf-8 -*-
+'''
+    Copyright (C) 2018 Gitcoin Core
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+'''
+from __future__ import unicode_literals
+
+import logging
+
+from django.conf import settings
+from django.db import models
+
+from dashboard.models import SendCryptoAsset
+from economy.models import SuperModel
+from eth_utils import to_checksum_address
+
+logger = logging.getLogger(__name__)
+
+
+class Token(SuperModel):
+    # Kudos specific fields -- lines up with Kudos contract
+
+    # Kudos Struct
+    price_finney = models.IntegerField()
+    num_clones_allowed = models.IntegerField(null=True, blank=True)
+    num_clones_in_wild = models.IntegerField(null=True, blank=True)
+    cloned_from_id = models.IntegerField()
+
+    # Kudos metadata from tokenURI
+    name = models.CharField(max_length=255)
+    description = models.CharField(max_length=510)
+    image = models.CharField(max_length=255, null=True)
+    rarity = models.CharField(max_length=255, null=True)
+    tags = models.CharField(max_length=255, null=True)
+    artist = models.CharField(max_length=255, null=True, blank=True)
+    platform = models.CharField(max_length=255, null=True, blank=True)
+
+    external_url = models.CharField(max_length=255, null=True)
+    background_color = models.CharField(max_length=255, null=True)
+
+    # Extra fields added to database (not on blockchain)
+    owner_address = models.CharField(max_length=255)
+    txid = models.CharField(max_length=255, null=True, blank=True)
+
+    def save(self, *args, **kwargs):
+        if self.owner_address:
```

Comment:
nit - method and class docstrings throughout, so code documentation is generated for all of this.